Makefile

```diff
@@ -308,7 +308,49 @@ go.cachedir:
 go.mod.cachedir:
 	@go env GOMODCACHE
 
-.PHONY: cobertura reviewable submodules fallthrough go.mod.cachedir go.cachedir run crds.clean $(TERRAFORM_PROVIDER_SCHEMA)
+DEP_CONSTRAINT ?= >= 0.0.0
+ifeq (-,$(findstring -,$(VERSION)))
+    DEP_CONSTRAINT = >= 0.0.0-0
+endif
+load-pkg: $(UP) build.all
```

Comment:
How about renaming the `SUBPACKAGES` make variable as `PROVIDERS`? The reasoning is `SUBPACKAGES` is a name with references to subgraphs and some preliminary theoretical work we did for the family providers. Instead of making that name more visible, it might be better to use a more user friendly name.